javascript/src/components/dashboard/DashboardList.jsx

```diff
@@ -0,0 +1,49 @@
+const React = require('react');
+const Alert = require('react-bootstrap').Alert;
+
+const Dashboard = require('./Dashboard');
+const EditDashboardModalTrigger = require('./EditDashboardModalTrigger');
+
+const PermissionsMixin = require('../../util/PermissionsMixin');
+
+const DashboardList = React.createClass({
+  mixins: [PermissionsMixin],
```

Comment:
Could you please add propTypes to the component? This way components are way easier to reuse.